Neo4j.Driver/Neo4j.Driver.Tests.TestBackend/Protocol/Time/FakeTime.cs

```diff
@@ -96,10 +106,32 @@ public void Freeze()
     public void Advance(int milliseconds)
     {
         _frozenTime = Now().AddMilliseconds(milliseconds);
+        foreach (var t in Timers)
+        {
+            t.Advance(milliseconds);
+        }
     }
 
     public void Unfreeze()
```

Comment:
The only thing I can see here is that freezing and unfreezing will have no effect on the timers, since they are always frozen. I like this solution to the timer problem, but perhaps it could be done with a stopwatch that gets started and stopped, in case testkit ever does something without freezing the time and expects the timers to tick?

(I know this is unlikely but if it did ever happen I think it might cause weird stuff that's hard to track down)